twinkle.js

```diff
@@ -69,6 +69,7 @@ Twinkle.defaultConfig = {
 	markRevertedPagesAsMinor: [ 'vand' ],
 	watchRevertedPages: [ 'agf', 'norm', 'vand', 'torev' ],
 	watchRevertedExpiry: 'yes',
+	customReversionSummaryList: [{value: 'Unsourced addition', label: 'Unsourced'}, {value: 'Unexplained removal of content', label: 'Removal of content'}, {value: 'Potential copyright violation', label: 'Copyright violation'}, {value: 'Disruptive editing', label: 'Disruptive editing'}],
```

Comment:
Not sure about this.  Would it make more sense to make this list be empty and allow users to provide it themselves, then offer a few options (such as these?) in addition to that list?  Kind of like the other customizations (warn/tag/welcome templates).  It'd be annoying if you want to remove one, but I think a dropdown would help with that (though see other comment).

Reply:
Agreed this looks a bit silly. Not so much agreed on the solution. I see that twinkleconfig is missing a "plainlist" data type that would have been the natural solution here. It might be worth adding that just for this usecase.

Reply:
We'd have to prepopulate it with something, no?

Reply:
There are two things to discuss here:

One is how to store the data.  A `plainlist` would be nice, but I don't think `customlist` is a huge hindrance.  The latter might make the display easier to parse for folks if there are multiple options.  So maybe "sources: removed reliable sources" and "sources: needs reliable sources" or something.  A plainlist does seem simpler given we just need one string of text.

The other is how to incorporate defaults, which I'm more concerned about.  With this setup, if someone customizes the list, they are forever left out of any changes we make unless they hit "reset, view the default options, then re-add from memory what they wanted; that's not good.  Instead we can do what is done elsewhere, namely include the basic uses in the module, and then add any items from `customReversionSummaryList` on top of that.  The advantage of the former is that users could cull the defaults, though.